### PR TITLE
feat(SCT-1500): screening decision loading and error state

### PR DESCRIPTION
**Pull request by JohnFarrellDev**
**What**  
When submitting a screening decision the submit button is now disabled.
If the API returns an error the error message is shown to the user.
We now have tests for ScreeningDecisionForm.

**Why**  
Tests for code reliability.
Rest to improve UX.

**Anything else?**

- Have you added any new third-party libraries? No
- Are any new environment variables or configuration values needed? No
- Anything else in this PR that isn't covered by the what/why? No

### CODE REVIEW

File: components/MashForms/ScreeningDecisionForm.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    ).toBeNull();
+
+    fireEvent.click(screen.getByText('Yes'));
+
+    expect(
+      screen.getByText('Please email your MASH manager about the urgent case.')
+    );
+  });
+
+  it('should disable submit button when submitScreeningDecision Submit button is clicked', () => {
```
Comment (Renny4Real) [inline components/MashForms/ScreeningDecisionForm.spec.tsx:50]: Does mean that the button is disabled after it is clicked? Why do we need to do this?

**Review by Renny4Real — APPROVED** [review]
Looks good. Just a question. 🎉

File: components/MashForms/ScreeningDecisionForm.spec.tsx (hunk 1 above)
Comment (JohnFarrellDev) [reply components/MashForms/ScreeningDecisionForm.spec.tsx:50]: We don't want a user clicking the button multiple times sending multiple API calls. 

It also indicates to the user that their submission is being sent as disabling the button modifies it's appearance.

Before a user could click, see no indication that clicking did anything,  click again. It wouldn't break anything but this is better UX and stops the API potentially getting spammed.

Comment (Renny4Real) [reply components/MashForms/ScreeningDecisionForm.spec.tsx:50]: I see 🙌🏾 . Do we show a notification/banner on the page? Might also help the user understand what's going on too?

**Pull request merged**